apps/ledger-live-mobile/src/screens/RequestAccount/02-SelectAccount.tsx

```diff
@@ -110,12 +110,12 @@ function SelectAccount({ navigation, route }: Props) {
   ) as { account: AccountLike; subAccount: SubAccount | null }[];
   const onSelect = useCallback(
     (account: AccountLike, parentAccount?: Account) => {
-      onSuccess && onSuccess(account, parentAccount);
```

Comment:
this is what caused the regression on https://ledgerhq.atlassian.net/browse/LIVE-7772 @kallen-ledger @sarneijim